### PR TITLE
[DDW-621] Fix the position in Japanese version of the Support page

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR changes the position of link in the Support Page for Japanese version of the page.

## Screenshots:

- Screenshot
Screenshot 2019-04-01 11 58 03

---

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`yarn run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn run flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `yarn run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn run storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ Changelog
 
 ## vNext
 
+### Fixes
```
Comment (nikolaglumac) [inline CHANGELOG.md]: @DeeJayElly please move the "Fixes" section below the "Features" 🙏

File: source/renderer/app/i18n/locales/ja-JP.json [hunk 2]
```diff
@@ -166,8 +166,8 @@
   "settings.support.reportProblem.downloadLogsProgressMessage": "ダウンロード用のログを準備しています",
   "settings.support.reportProblem.downloadLogsSuccessMessage": "ログのダウンロードに成功しました。",
   "settings.support.reportProblem.linkUrl": "https://iohk.zendesk.com/hc/ja/requests/new/",
-  "settings.support.steps.downloadLogs.description": "サポートチームが問題を調査しやすくするために、ここからログをダウンロードして、ダウンロードしたファイルをサポートリクエストの送信時に添付してください。ログに{stepsDownloadLogsLink}",
-  "settings.support.steps.downloadLogs.link": "機密情報は含まれていません。",
+  "settings.support.steps.downloadLogs.description": "サポートチームが問題を調査しやすくするために、{stepsDownloadLogsLink}して、ダウンロードしたファイルをサポートリクエストの送信時に添付してください。ログに機密情報は含まれていません。",
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/ja-JP.json]: @DeeJayElly are you sure we don't need a space after `{stepsDownloadLogsLink}` part?

Comment (DeeJayElly) [reply source/renderer/app/i18n/locales/ja-JP.json]: Hmm i was just looking at the screenshot from the task and don't think we have a space before or after the {stepsDownloadLogsLink}?

Comment (nikolaglumac) [reply source/renderer/app/i18n/locales/ja-JP.json]: Alright then :)

File: CHANGELOG.md (hunk 1 above)
Comment (DeeJayElly) [reply CHANGELOG.md]: Updated

**Pull request merged**